src/main/kotlin/me/serce/solidity/ide/run/SolidityRunConfig.kt

```diff
@@ -0,0 +1,199 @@
+package me.serce.solidity.ide.run
+
+import com.intellij.diagnostic.logging.LogConfigurationPanel
+import com.intellij.execution.*
+import com.intellij.execution.configuration.EnvironmentVariablesComponent
+import com.intellij.execution.configurations.*
+import com.intellij.execution.filters.TextConsoleBuilderFactory
+import com.intellij.execution.runners.ExecutionEnvironment
+import com.intellij.execution.util.JavaParametersUtil
+import com.intellij.execution.util.ProgramParametersUtil
+import com.intellij.openapi.module.Module
+import com.intellij.openapi.module.ModuleManager
+import com.intellij.openapi.options.SettingsEditor
+import com.intellij.openapi.options.SettingsEditorGroup
+import com.intellij.openapi.options.ShowSettingsUtil
+import me.serce.solidity.ide.run.ui.SolidityConfigurableEditorPanel
+import me.serce.solidity.ide.settings.SoliditySettings
+import me.serce.solidity.ide.settings.SoliditySettingsConfigurable
+import me.serce.solidity.lang.psi.SolContractDefinition
+import me.serce.solidity.lang.psi.SolFunctionDefinition
+import org.jdom.Element
+import java.util.*
+
+
+class SolidityRunConfig(configurationModule: SolidityRunConfigModule?, factory: ConfigurationFactory?) : ModuleBasedConfiguration<SolidityRunConfigModule>(configurationModule, factory), CommonJavaRunConfigurationParameters {
+
+  var myData: Data = Data()
+
+  override fun getEnvs(): Map<String, String> {
+    return myData.envs;
+  }
+
+  override fun setAlternativeJrePath(p0: String?) {
```

Comment:
I believe it would be beneficial to give parameters readable names

Reply:
Done